ui/client/components/graph/node-modal/editors/expression/Editor.ts

```diff
@@ -84,7 +85,7 @@ export const editors: Record<EditorType, EditorConfig> = {
   },
   [EditorType.STRING_PARAMETER_EDITOR]: {
     ...defaults,
-    editor: () => StringEditor,
+    editor: (param, displayRawEditor, expressionObj) => expressionObj?.language === "spel" ?  SpelStringEditor : StringEditor,
```

Comment:
Please add more generic support for other editors by passing language

Reply:
Done